Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -1479,12 +1479,12 @@ public async Task<string> GetWanApiUrl(CancellationToken cancellationToken)
                     LogErrorResponseBody = false,
                     LogErrors = false,
                     LogRequest = false,
-                    TimeoutMs = 10000,
```

Comment:
why remove?

Reply:
Can't set the timeout for each request, only per httpclient.